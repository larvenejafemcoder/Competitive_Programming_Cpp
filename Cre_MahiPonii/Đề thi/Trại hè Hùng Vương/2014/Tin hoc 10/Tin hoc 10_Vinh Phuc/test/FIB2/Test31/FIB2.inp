55
36 7310490
12 124
34 4042431
34 7300409
35 8929674
40 11086215
17 81
23 10214
38 27313635
10 54
37 28914468
15 844
16 728
33 5133605
35 8350050
23 32823
23 27249
34 6410754
9 20
35 7313802
31 138255
16 352
20 4123
35 8809405
3 2
44 615271012
2 2
9 18
7 17
4 1
38 11281501
29 244405
24 19451
22 14048
14 253
40 107568221
3 1
8 5
34 6021939
6 8
23 12883
17 2524
2 1
20 10082
24 7348
45 1511144053
14 228
4 1
33 5249122
24 8734
40 35569680
17 1252
21 2274
7 4
9 15
