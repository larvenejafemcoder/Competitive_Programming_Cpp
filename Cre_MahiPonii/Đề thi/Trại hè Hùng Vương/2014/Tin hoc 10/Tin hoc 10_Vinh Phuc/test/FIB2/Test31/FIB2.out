2792359
47
1544071
2788508
3410832
4234557
31
3902
10432880
21
11044344
322
278
1960863
3189435
12537
10408
2448690
8
2793624
52809
135
1575
3364893
1
235012614
1
7
7
1
4309150
93355
7430
5366
97
41087404
0
2
2300176
3
4921
964
1
3851
2807
577205666
87
1
2004986
3336
13586409
478
869
2
6
